sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentModelAdministrationClient/DocumentModelAdministrationLiveTests.cs

```diff
@@ -176,170 +176,237 @@ public void BuildModelError()
-            var targetClient = CreateDocumentModelAdministrationClient(useTokenCredential);
+            var client = CreateDocumentModelAdministrationClient(useTokenCredential);
+            var sourceModelId = Recording.GenerateId();
             var modelId = Recording.GenerateId();
+            var description = $"This model was generated by a .NET test: {nameof(CopyDocumentModelTo)}";
+            var tags = new Dictionary<string, string>() { { "tag1", "value1" }, { "tag2", "value2" } };
+            var startTime = Recording.UtcNow;
 
-            await using var trainedModel = await BuildDisposableDocumentModelAsync(modelId);
+            await using var disposableModel = await BuildDisposableDocumentModelAsync(sourceModelId);
 
-            var targetModelId = Recording.GenerateId();
-            DocumentModelCopyAuthorization targetAuth = await targetClient.GetCopyAuthorizationAsync(targetModelId);
+            DocumentModelCopyAuthorization copyAuthorization = await client.GetCopyAuthorizationAsync(modelId, description, tags);
+            CopyDocumentModelToOperation operation = null;
 
-            CopyDocumentModelToOperation operation = await sourceClient.CopyDocumentModelToAsync(WaitUntil.Completed, trainedModel.ModelId, targetAuth);
+            try
+            {
+                operation = await client.CopyDocumentModelToAsync(WaitUntil.Completed, sourceModelId, copyAuthorization);
+            }
+            finally
+            {
+                if (operation != null && operation.HasValue)
+                {
+                    await client.DeleteDocumentModelAsync(modelId);
+                }
+            }
 
             Assert.IsTrue(operation.HasValue);
 
-            DocumentModelDetails copiedModel = operation.Value;
-
-            ValidateDocumentModelDetails(copiedModel);
-            Assert.AreEqual(targetAuth.TargetModelId, copiedModel.ModelId);
-            Assert.AreNotEqual(trainedModel.ModelId, copiedModel.ModelId);
+            DocumentModelDetails sourceModel = disposableModel.Value;
+            DocumentModelDetails model = operation.Value;
 
-            Assert.AreEqual(1, copiedModel.DocumentTypes.Count);
-            Assert.IsTrue(copiedModel.DocumentTypes.ContainsKey(modelId));
+            Assert.AreEqual(modelId, model.ModelId);
+            Assert.AreEqual(description, model.Description);
+            Assert.AreEqual(ServiceVersionString, model.ApiVersion);
+            Assert.Greater(model.CreatedOn, startTime);
+            Assert.Greater(model.ExpiresOn, model.CreatedOn);
 
-            DocumentTypeDetails documentType = copiedModel.DocumentTypes[modelId];
+            CollectionAssert.AreEquivalent(tags, model.Tags);
 
-            Assert.AreEqual(DocumentBuildMode.Template, documentType.BuildMode);
+            AssertDocumentTypeDictionariesAreEquivalent(sourceModel.DocumentTypes, model.DocumentTypes);
         }
 
+        #endregion Copy
+
+        #region Compose
```

Comment:
The following tests can be a bit hard to review since git can't figure out how old tests map to the new ones (there's no 1:1 parity since some of them have been merged).